src/data/actions/app.js

```diff
@@ -12,11 +12,19 @@ const closeModal = createAction('closeModal');
  * @param {string} courseId - course ID from app context
  */
 const setCourseId = createAction('setCourseId');
+
+const filterMenu = StrictDict({
+  endTransition: createAction('filterMenu/endTransition'),
```

Comment:
Is there a reason to action on start/end open? It seems needlessly actiony...

Reply:
because of how is called in the component.  All is for simpler components.